invoice/src/main/java/org/killbill/billing/invoice/dao/InvoiceDaoHelper.java

```diff
@@ -331,6 +334,32 @@ public boolean apply(final InvoiceModelDao input) {
         }
     }
 
+    private void setTrackingIdsFromTransaction(final Iterable<InvoiceModelDao> invoices, final EntitySqlDaoWrapperFactory entitySqlDaoWrapperFactory, final InternalTenantContext context) {
+        final InvoiceTrackingSqlDao invoiceTrackingidSqlDao = entitySqlDaoWrapperFactory.become(InvoiceTrackingSqlDao.class);
+        final List<InvoiceTrackingModelDao> trackingIds = invoiceTrackingidSqlDao.getByAccountRecordId(context);
```

Comment:
Not suggesting to change this for now, but a bit worried about memory usage...

Reply:
Ack - will pull info for the matching invoices.